lochness/rpms/__init__.py

```diff
@@ -178,7 +183,7 @@ def sync(Lochness, subject, dry=False):
             os.chmod(dirname, 0o0755)
             source_df.to_csv(target_df_loc, index=False)
             os.chmod(target_df_loc, 0o0755)
-            process_and_copy_db(Lochness, subject, target_df_loc, proc_dst)
+            # process_and_copy_db(Lochness, subject, target_df_loc, proc_dst)
```

Comment:
Why disabled?

Reply:
This was for PII removal, and we have moved these functionalities to dpanonymize.